docs/pages/getting-started/quick-start.md

```diff
@@ -23,6 +23,66 @@ dojoup
 
 For full `dojoup` reference and debugging see [Dojoup](/toolchain/dojoup.md).
 
+### Install Scarb (optional)
+
+To ensure the proper functioning of the language server, your local `Scarb` version must be equal to or higher than the Dojo's Cairo version. To accomplish this, first check your Dojo's Cairo version. To achieve this, run the following command:
+
+```bash
+sozo --version
+```
+
+You might expect an output similar to this:
+
+```console
+sozo 0.6.0-alpha.9
+scarb: 2.5.4
+cairo: 2.5.4
+sierra: 1.4.0
+```
+
+After that, check your local `Scarb` version:
+
+```sh
+scarb --version
+```
+
+If your local `Scarb` version is lower than the Dojo's Cairo version, you have to install `Scarb`.
+
+To accomplish it, please refer to the [installation instructions](https://docs.swmansion.com/scarb/download).
+We strongly recommend that you install
+Scarb via [asdf](https://docs.swmansion.com/scarb/download.html#install-via-asdf), a CLI tool that can manage
+multiple language runtime versions on a per-project basis.
+This will ensure that the version of Scarb you use to work on a project always matches the one defined in the
+project settings, avoiding problems related to version mismatches.
+
+Please refer to the [asdf documentation](https://asdf-vm.com/guide/getting-started.html) to install all
+prerequisites.
+
+Once you have `asdf` installed locally, you can download Scarb plugin with the following command:
+
+```bash
+asdf plugin add scarb
+```
+
+This will allow you to download specific versions. You can choose the same version as the Dojo's Cairo version, for example, 2.5.4, with the following command:
```

Comment:
```suggestion
This will allow you to download specific versions. You can choose the same version as the Dojo's Cairo version, for example, `2.5.4`, with the following command:
```